src/js/disability-benefits/526EZ/config/form.js

```diff
@@ -230,7 +232,56 @@ const formConfig = {
         },
         // pageFour: {},
         // pageFive: {},
-        // pageSix: {},
+        privateRecordChoice: {
+          title: '',
+          path: 'supporting-evidence/:index/private-medical-records-choice',
+          showPagePerItem: true,
+          arrayPath: 'disabilities',
+          depends: (formData, index) => _.get(`disabilities.${index}.view:privateMedicalRecords`, formData),
+          uiSchema: {
+            disabilities: {
+              items: {
+                'ui:title': disabilityNameTitle,
+                'ui:description': privateRecordsChoice,
+                'view:uploadPrivateRecords': {
+                  'ui:title': 'Do you want to upload your private medical records?',
+                  'ui:widget': 'radio',
+                  'ui:options': {
+                    labels: {
+                      yes: 'Yes',
+                      no: 'No, please get them from my doctor'
+                    }
+                  }
+                },
+                'view:privateRecordsChoiceHelp': {
+                  'ui:description': privateRecordsChoiceHelp
+                }
+              }
+            }
+          },
+          schema: {
+            type: 'object',
+            properties: {
+              disabilities: {
```

Comment:
Is the backend schema planned to be used here later?

Reply:
So this is where it gets a little tricky. I think `formData.disabilities` is going to be filled piecemeal across multiple pages, so when we use the schema from `vets-json-schema`, it'll also have to be piecemeal across different pages. The fields here are `view:`-only, though, so it doesn't make sense to use the backend schema yet.

Actually, I have a hunch that none of the data in chapter 4 is supposed to actually belong in the `disabilities` property, but we have to populate the items in `disabilities` because we're iterating over them. Later, in our `transform()`, we'll pull them out and put them where they belong.

But to answer your question, yes, I think we will use the backend schema when we build out one of the preceding chapters. But not yet.

Reply:
Thanks for the context @cvalarida.